controllers/foodsController.js

```diff
@@ -30,6 +30,35 @@ const index = async (req, res) => {
   }
 }
 
+const show = async (req, res) => {
+  try {
+    const food = await Food.findOne({
+      where: {
+        id: req.params.id
+      }
+    });
+    res.setHeader("Content-Type", "application/json");
+    res.status(200).send(JSON.stringify(food));
+  } catch (error) {
+    res.setHeader("Content-Type", "application/json");
+    res.status(404).send({error});
+  }
+}
+
+const update = async (req, res) => {
+  try {
+    let food = await Food.findOne({where: {id: req.params.id}})
+    food.name = req.body.food_name
+    food.calories = req.body.calories
+    food.save({fields: ['name', 'calories']})
+    res.setHeader("Content-Type", "application/json");
+    res.status(200).send(JSON.stringify(food));
+  } catch (error) {
+    res.setHeader("Content-Type", "application/json");
+    res.status(404).send({error});
+  }
+}
+
 module.exports = {
-  index, add
+  index, show, add, update
```

Comment:
@PeregrineReed What's add? Should that be create?

Reply:
`create` is a reserved word, I wish it could use it but sequelize beat me to it 😞